drivers/wifi/nrf700x/Kconfig

```diff
@@ -241,6 +241,10 @@ config NRF700X_SR_COEX
 	bool "Enable Wi-Fi and SR coexistence support"
 	def_bool $(dt_nodelabel_enabled,nrf_radio_coex)
 
+config NRF700X_THREAD_COEX
```

Comment:
This should be before the SML change as it depends on this, but anyways we don't need this change as suggested above.

Reply:
The Config is updated and the commit is moved down accordingly